invenio_app_ils/records_relations/api.py

```diff
@@ -178,11 +178,11 @@ def _build_sibling(self, related_pid, relation_type):
 
         # add also title, language and edition of the sibling
         r["title"] = sibling.get("title", "")
-        language = sibling.get('language')
-        if language:
-            r["language"] = language
+        language = sibling.get('languages')
+        if language and relation_type == 'language':
```

Comment:
I think I didn't add this part of the if `relation_type == 'language'` on purpose but we can keep it for sure.
This is because if we want to display in the UI the language of the other doc, even if we are editing the `edition` relation for example, we have the info.

Reply:
I will remove it then